Makefile

```diff
@@ -17,20 +17,35 @@ all: build run
 run:
 	./${OUT}
 
-build: go/generate go/format
+run/parallel:
+	cat config/config.json | jq -r '.peers | .[].name' | parallel ./build/rfs -n {} | jq
+
+build:
 	mkdir -p build
 	go build -ldflags "${LDFLAGS}" -o ${OUT} main.go
 
+build-full: go/generate go/format go/verify go/test build
+
 go/generate:
 	go generate ./...
 
 go/format:
 	go fmt ./...
-	#goimports ./..
+	#@goimports ./.. >/dev/null
 
 go/test:
 	go test -v -race ./...
 
+go/verify:
+	go vet ./...
+
+go/protobuf:
+	@find ./protobuf -type f -name *.proto -printf "%f\n" | \
+		xargs -I{} protoc {}\
+			--proto_path=protobuf \
+			--go_opt=module=github.com/Zamerykanizowana/replicated-file-system \
+			--go_out=Mprotobuf/{}=:.
+
```

Comment:
Do we need to add the output path to `.gitignore`?

Reply:
we can't, the generated protobufs will be part of the repo, they will be imported and used by other packages

Reply:
Oh, fair enough!